node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
If Johnny-Five has set precedence here I'm all for following it, consistency is awesome!